docs/artefacts/offramps.md

```diff
@@ -966,11 +966,39 @@ lost.
 Example:
 
 ```yaml
-onramp:
+offramp:
   - id: otlp
     type: otel
     codec: json
     config:
       port: 4317
       host: 10.0.2.1
 ```
+
+# gcs
```

Comment:
It would be nice to get an overview of how to send the different payloads to the offramp, you could look at `kv` or `dns` as an inspiration for how command-driven offramps usually are documented :) (those are not perfect so if you have a better idea we'll be  more then happy!)